mqtt/mqtt-bridge/src/persist/waking_state/ring_buffer/block.rs

```diff
@@ -27,32 +31,37 @@ pub const BLOCK_HINT: u32 = 0xdead_beef;
 /// validation.
 #[derive(Copy, Clone, Debug, Deserialize, Hash, Serialize)]
 #[repr(C)]
-pub(crate) struct BlockHeader {
+pub(crate) struct BlockHeaderV1 {
```

Comment:
Rename to V1 for versioning. Let me know if a different versioning format would be preferable!